src/main/java/exercise31/KarvonenHeartRate.java

```diff
@@ -2,6 +2,7 @@
 
 public class KarvonenHeartRate {
 
+    //Data type?
```

Comment:
So this variable is called `table` but seems to only hold `int`s. Is the data-type wrong or is it being used for something else?